src/components/MovieDetails/MovieDetails.js

```diff
@@ -2,11 +2,68 @@ import React from 'react';
 import './MovieDetails.scss';
 import { trailers } from '../../trailers';
 
-const MovieDetails = ({ location }) => {
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { setMovies, setRatings } from '../../actions/actions';
+import { addUserRating, getRatings, deleteRating } from '../../apiCalls';
+
+
+export const MovieDetails = ({ location, user, ratings, setRatings }) => {
+
+  const removeRating = async (event) => {
+    let ratingID = event.target.id;
+    await deleteRating(user.id, ratingID)
+    await getSetRatings()
+  }
+
+  const checkIfRated = (ratings, movie) => {
+    let movieRating = ratings.find(rating => {
```

Comment:
This doesn't look like it's being redefined anywhere, so it should be a `const`.